components/post_view/post_body_additional_content/post_body_additional_content.tsx

```diff
@@ -4,16 +4,17 @@
 import {Post, PostEmbed} from 'mattermost-redux/types/posts';
 
 import {getEmbedFromMetadata} from 'mattermost-redux/utils/post_utils';
+import {PostEmbed as AppEmbed} from 'mattermost-redux/types/apps';
```

Comment:
Is `PostEmbed` being used?

Reply:
In this component is used, yes. PostEmbed is a completely different structure. I would be more than willing to change the name for the new "PostEmbed", but I am lacking of imagination for the names :sweat_smile: